src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -3993,7 +3989,8 @@ namespace EnergyPlus::ZoneEquipmentManager {
 
                 // Calculate standard return air flow rate using default method of inlets minus exhausts adjusted for "balanced" exhaust flow
                 StdTotalReturnMassFlow = TotInletAirMassFlowRate + ZoneMixingNetAirMassFlowRate -
-                                         (TotExhaustAirMassFlowRate - state.dataZoneEquip->ZoneEquipConfig(ZoneNum).ZoneExhBalanced);
+                    (TotExhaustAirMassFlowRate - state.dataZoneEquip->ZoneEquipConfig(ZoneNum).ZoneExhBalanced);
+
```

Comment:
Reverted back to the original implementation since the exhaust balanced flow schedule is ignored at the get input if not needed.